api/tests/Unit/PoolCandidatePolicyTest.php

```diff
@@ -0,0 +1,281 @@
+
+        $this->teamPool = Pool::factory()->create([
+            'user_id' => $this->poolOperatorUser->id,
+            'team_id' => $this->team->id,
+        ]);
+
+        $this->poolCandidate = PoolCandidate::factory()->create([
+            'user_id' => $this->candidateUser->id,
+            'pool_id' => $this->teamPool->id
+        ]);
+    }
+
+
+    /**
+     * Assert that no user may view any pool candidate
+     *
+     * @return void
+     */
+    public function testViewAny()
+    {
+        $this->assertFalse($this->guestUser->can('viewAny', PoolCandidate::class));
+        $this->assertFalse($this->applicantUser->can('viewAny', PoolCandidate::class));
+        $this->assertFalse($this->candidateUser->can('viewAny', PoolCandidate::class));
+        $this->assertFalse($this->poolOperatorUser->can('viewAny', PoolCandidate::class));
+        $this->assertFalse($this->requestResponderUser->can('viewAny', PoolCandidate::class));
+        $this->assertFalse($this->adminUser->can('viewAny', PoolCandidate::class));
+    }
+
+    /**
+     * Assert that only owner can view draft pool candidates
+     *
+     * @return void
+     */
+    public function testViewDraft()
+    {
+        $this->poolCandidate->submitted_at = null;
+        $this->poolCandidate->pool_candidate_status = ApiEnums::CANDIDATE_STATUS_DRAFT;
+        $this->poolCandidate->save();
+
+        $this->assertTrue($this->candidateUser->can('view', $this->poolCandidate));
+
+        $this->assertFalse($this->guestUser->can('view', $this->poolCandidate));
+        $this->assertFalse($this->applicantUser->can('view', $this->poolCandidate));
+        $this->assertFalse($this->poolOperatorUser->can('view', $this->poolCandidate));
+        $this->assertFalse($this->requestResponderUser->can('view', $this->poolCandidate));
+        $this->assertFalse($this->adminUser->can('view', $this->poolCandidate));
+    }
+
+    /**
+     * Assert that the following can view a submitted pool candidate:
+     *
+     * owner, pool operator, request responder, admin
+     *
+     * @return void
+     */
+    public function testViewSubmitted()
+    {
+        $this->poolCandidate->submitted_at = config('constants.past_date');
+        $this->poolCandidate->pool_candidate_status = ApiEnums::CANDIDATE_STATUS_NEW_APPLICATION;
+        $this->poolCandidate->save();
+
+        $this->assertTrue($this->candidateUser->can('view', $this->poolCandidate));
+        $this->assertTrue($this->poolOperatorUser->can('view', $this->poolCandidate));
+        $this->assertTrue($this->requestResponderUser->can('view', $this->poolCandidate));
+        $this->assertTrue($this->adminUser->can('view', $this->poolCandidate));
+
+        $this->assertFalse($this->guestUser->can('view', $this->poolCandidate));
+        $this->assertFalse($this->applicantUser->can('view', $this->poolCandidate));
+    }
+
+    /**
+     * Assert that only guest cannot create a draft pool candidate
```

Comment:
Aren't only Applicants allowed to do this?

Reply:
Oops, I was assigning base roles to all users so had to check all. Fixed this in [improve policy checks and make test more strict](https://github.com/GCTC-NTGC/gc-digital-talent/pull/5815/commits/594bfd83547582f3b67a6967241f4ebacd3b18f0)